[feature][website] release notes for pulsar manager 0.3.0 (#15772)
apache/pulsar: site2/website/pulsar-manager-release-notes.md

## Apache Pulsar Manager

### 0.3.0 &mdash; 2022-05-25 <a id="0.3.0"></a>

> * Fixed mysql schema field token error [PR-339](https://github.com/apache/pulsar-manager/pull/339)

Reviewer: ```suggestion
* Fixed MySQL schema field token error [PR-339](https://github.com/apache/pulsar-manager/pull/339)
```

1. `Fix` of `Fixed`? Keep tense consistent overall. For example, use past tense or simple tense for all verbs. Please update all occurrences. 

3. Capitalize proper nouns correctly.  https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.je5thtkj42cb

Author: fixed, thanks.

> * Fixed topics List page, search Topics bug [PR-342](https://github.com/apache/pulsar-manager/pull/342)

Reviewer: ```suggestion
* Fix the bugs in topics list page and search topics [PR-342](https://github.com/apache/pulsar-manager/pull/342)
```
Do you mean this？

Author: fixed, thanks.

> * Fixed pulsar manager's swagger dose not work. [PR-369](https://github.com/apache/pulsar-manager/pull/369)

Reviewer: ```suggestion
* Fix the issue of Swagger does not work. [PR-369](https://github.com/apache/pulsar-manager/pull/369)
```

Author: fixed, thanks.

* Add offload threshold input [PR-375](https://github.com/apache/pulsar-manager/pull/375)
* Add dashboard service for aggregated information collection [PR-326](https://github.com/apache/pulsar-manager/pull/326)

> * Multi bookie cluster Support [PR-388](https://github.com/apache/pulsar-manager/pull/388)

Reviewer: ```suggestion
* Add multi bookie cluster support [PR-388](https://github.com/apache/pulsar-manager/pull/388)
```
do you mean this?

Author: fixed, thanks.

> * Fixed the bug that admin roles cannot be deleted. [PR-397](https://github.com/apache/pulsar-manager/pull/397)

Reviewer: Use a period or not? please keep consistent

Author: fixed, thanks.

> * Add filterable to el-select enables filtering for tenant/namespace/topic [PR-440](https://github.com/apache/pulsar-manager/pull/440)

Reviewer: filterable is an adjective. filterable that? filterable condition?

Author: fixed, thanks.

* Updated to fix Log4J security vulnerabilities [PR-438](https://github.com/apache/pulsar-manager/pull/438)
* Allow user to assign tenant as resource to role [PR-436](https://github.com/apache/pulsar-manager/pull/436)

> * Add support for casdoor [PR-446](https://github.com/apache/pulsar-manager/pull/446)

Reviewer: ```suggestion
* Add support for Casdoor [PR-446](https://github.com/apache/pulsar-manager/pull/446)
```

Author: fixed, thanks.
